evm-node.md

```diff
@@ -0,0 +1,304 @@
+EVM Integration with Production Client
+======================================
+
+Contained in this file is glue code needed in order to enable the ability to use KEVM as a VM for an actual Ethereum node.
+
+```{.k .node}
+requires "evm.md"
+
+module EVM-NODE
+    imports EVM
+    imports K-REFLECTION
+    imports COLLECTIONS
+    imports BYTES
```

Comment:
We should also import `EVM-OPTIMIZATIONS` so we get those speedups for long-running executions. It is in file `optimizations.md`